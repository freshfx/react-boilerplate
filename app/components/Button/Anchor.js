import styled from '@emotion/styled'

import buttonStyles from './buttonStyles'

const Anchor = styled.a`
  ${buttonStyles};
`

export default Anchor
