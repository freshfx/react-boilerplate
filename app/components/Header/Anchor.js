import styled from 'styled-components'
import NormalA from 'components/Anchor'

const Anchor = styled(NormalA)`
  padding: 2em 0;
`

export default Anchor
