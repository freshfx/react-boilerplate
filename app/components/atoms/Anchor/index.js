/**
 * A link to a certain page, an anchor tag
 */

import styled from '@emotion/styled'

const Anchor = styled.a`
  color: #41addd;

  &:hover {
    color: #6cc0e5;
  }
`

export default Anchor
