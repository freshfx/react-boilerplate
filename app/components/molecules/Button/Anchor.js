import styled from '@emotion/styled'

import buttonStyles from 'components/atoms/Button/buttonStyles'

const Anchor = styled.a`
  ${buttonStyles};
`

export default Anchor
