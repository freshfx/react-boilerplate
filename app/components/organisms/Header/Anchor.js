import styled from '@emotion/styled'

import NormalA from 'components/atoms/Anchor'

const Anchor = styled(NormalA)`
  padding: 2em 0;
`

export default Anchor
