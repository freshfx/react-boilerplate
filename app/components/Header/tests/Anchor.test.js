import React from 'react'
import {render} from '@testing-library/react'

import Anchor from '../Anchor'

describe('Anchor', () => {
  it('should match snapshot', () => {
    const {container} = render(<Anchor />)
    expect(container.firstChild).toMatchSnapshot()
  })
})
