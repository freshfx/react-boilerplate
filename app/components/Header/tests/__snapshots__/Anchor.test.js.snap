// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<A /> should match the snapshot 1`] = `
.c1 {
  color: #41addd;
}

.c1:hover {
  color: #6cc0e5;
}

.c0 {
  padding: 2em 0;
}

<a
  className="c0 c1"
/>
`;
